metrics/brier_score/brier_score.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Brier Score Metric"""
+
+import datasets
+from sklearn.metrics import brier_score_loss
+import evaluate
+
+
+_CITATION = """\
+@article{scikit-learn,
+ title={Scikit-learn: Machine Learning in {P}ython},
+ author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.
+         and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.
+         and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and
+         Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},
+ journal={Journal of Machine Learning Research},
+ volume={12},
+ pages={2825--2830},
+ year={2011}
+}
+"""
+
+_DESCRIPTION = """\
+Brier score is a type of evaluation metric for classification tasks, where you predict outcomes such as win/lose, spam/ham, click/no-click etc.
+`BrierScore = 1/N * sum( (p_i - o_i)^2 )`
+"""
+
+
+_KWARGS_DESCRIPTION = """
+Args:
+    predictions: Ground truth (correct) target values. shape = [n_samples]
+    references: Estimated probabilities. shape = [n_samples]
+    sample_weight: Sample weights.
+    pos_label: The label of the positive class.
+    
+Returns:
+    The Brier score.
+    
+Examples:
+    Example-1: if y_true in {-1, 1} or {0, 1}, pos_label defaults to 1;
```

Comment:
The current tests fail because in the examples `np` is not defined: can you add an import?

Reply:
I fixed it.